content/en/notebooks/_index.md

```diff
@@ -153,19 +153,40 @@ In a Notebook cell, click **Edit** to view the cell configuration in edit mode.
 
 #### Types of content
 
-Notebooks support visualizations and text cells. Text cells are formatted with [Markdown][5], which enables the use of headings, subheadings, links, images, lists, and code blocks. Notebooks also support diagrams formatted with [MermaidJS][15].
+Notebooks support visualizations and text cells. Text cells are formatted with [Markdown][5], which enables the use of headings, subheadings, links, images, lists, and code blocks. Notebooks also support diagrams formatted with [MermaidJS][6].
 
 Graphs in notebooks support all Datadog data sources: metrics, log events, Indexed Spans, live processes, network traffic, RUM events, profiling metrics, security signals, and more. Graphs are created with the Datadog query editor. Notebooks support:
 
-* [Timeseries][6]
-* [Top List][7]
-* [Table][8]
-* [Heatmap][9]
-* [Distribution][10]
-* [List][11]
-* [Query value][12]
-* [Funnel][13]
-* [Pie][14]
+* [Timeseries][7]
+* [Top List][8]
+* [Table][9]
+* [Heatmap][10]
+* [Distribution][11]
+* [List][12]
+* [Query value][13]
+* [Funnel][14]
+* [Pie][15]
+
+#### Limit edit access
```

Comment:
Can you move this section up so that it's under **Notebook Configuration** rather than **Cell Configuration**?